### PR TITLE
Images

### PR DESCRIPTION
**Pull request by mitchdowney**

### CODE REVIEW

File: src/lib/navigate.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      })
+    ],
+    key: 'Podcasts'
+  })
+
+  navigation.dispatch(resetAction)
+}
+
+export const navigateToPodcastScreenWithPodcast = (navigation: any, podcast: any, viewType: string) => {
+  navigation.dismiss()
```
Comment (kreonjr) [inline src/lib/navigate.ts:61]: Is this needed here? Is reset not cutting it?

File: src/screens/SettingsScreen.tsx [hunk 2]
```diff
@@ -342,30 +355,30 @@ export class SettingsScreen extends React.Component<Props, State> {
...
-              <View>
-                <Divider style={styles.divider} />
-                <Button
-                  isWarning={true}
-                  onPress={this._handleToggleDeleteAccountDialog}
-                  text={translate('Delete Account')}
-                  wrapperStyles={styles.button}
-                />
-              </View>
+              <Button
```
Comment (kreonjr) [inline src/screens/SettingsScreen.tsx:376]: Is this button needed in Podable?

File: src/lib/navigate.ts (hunk 1 above)
Comment (mitchdowney) [reply src/lib/navigate.ts:61]: I needed to use the `navigation.dismiss()` to get the modal to dismiss itself. Without calling dismiss, the modal would stay and the stack below wouldn't navigate.

File: src/screens/SettingsScreen.tsx (hunk 2 above)
Comment (mitchdowney) [reply src/screens/SettingsScreen.tsx:376]: No, good catch...how about we add a DISABLE_LOGIN env var and use it disable this and all logged in user features?

Comment (kreonjr) [reply src/screens/SettingsScreen.tsx:376]: Yes I think I agree. Better to make it configurable than not :)

File: src/lib/navigate.ts (hunk 1 above)
Comment (kreonjr) [reply src/lib/navigate.ts:61]: Ok gotcha. I'm a bit concerned with timing issues here but if its working fine, no problem.

**Pull request merged**